src/core/io/pla_parser.cpp

```diff
@@ -81,6 +83,8 @@ namespace syrec {
 
         parsePla(tt, is);
 
+        extend(tt);
```

Comment:
Is this the only place where the `extend` function is used now?
If yes, then consider moving it here.